tox.ini

```
@@ -98,6 +99,10 @@ deps=
     wheel
     build
     nox
+    {importer-wandb}: pydantic>=2
+    {importer-wandb}: polars
+    {importer-wandb}: rich
+    {importer-wandb}: filelock
```

```suggestion
    importer: pydantic>=2
    importer: polars
    importer: rich
    importer: filelock
```

should be enough and will work for when you add the mlflow